cumulus_library/studies/vocab/.gitignore

```diff
@@ -0,0 +1 @@
+icd/*.parquet
```

Comment:
i put this in the study folder just in case we ever decide to break it off into its own repo